src/ThemeProvider.js

```diff
@@ -0,0 +1,89 @@
+import React, {PropTypes, Component} from 'react'
+import mitt from 'mitt'
+
+export const CHANNEL = '__glamorous__'
```

Comment:
I'd actually recommend against the Symbol approach, having just removed that exact setup from Preact.  Most people will end up getting a Symbol polyfill in their bundle (which includes polyfillying `typeof`, which is slow).  Just use an underscore key like you have there.

FWIW symbols are supported in context, just unnecessary. Also fun caveat: with the string you can have 2 copies of this lib and they will talk properly. With the Symbol, if you bundle twice and get 2 polyfills, they may not have the same generated key.